kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h

```diff
@@ -2149,10 +2144,9 @@ class ResidualBasedEliminationBuilderAndSolverWithConstraints
 
         // Filling constant vector
         if (ComputeConstantVector) {
-            #pragma omp parallel for
-            for (int i = 0; i < static_cast<int>(BaseType::mEquationSystemSize); ++i) {
-                rConstantVector[i] = 0.0;
-            }
+            IndexPartition<std::size_t>(BaseType::mEquationSystemSize).for_each([&rConstantVector](std::size_t Index){
```

Comment:
```suggestion
            IndexPartition<std::size_t>(BuilderAndSolverBaseType::mEquationSystemSize).for_each([&rConstantVector](std::size_t Index){
```

can you try with this?

Reply:
Its working fine on local machine, 
but on CentOS giving error :
`ool)':
/__w/Kratos/Kratos/kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h:2147:40: error: invalid declaration of 'Kratos::BuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver>::mEquationSystemSize'
             IndexPartition<std::size_t>(BuilderAndSolverBaseType::mEquationSystemSize).for_each([&rConstantVector](std::size_t Index){`

Reply:
I saw

as a last test, can you try with `this->`?

Reply:
okay ..

Reply:
worked !!!!Thanks :)